### DISCUSSION TITLE
Github Discussions are the new forums

### DISCUSSION THREAD

**Discussion by ThomasFOG** [MonoGame/MonoGame#8130] [Announcements]
Hello there,

We would like to move away from the [forums based on Discourse](https://community.monogame.net/).

While the Discourse forum is still available and functioning, we would like to encourage you to move over Github Discussions.

The Discourse forum will at some point retire and close. We are currently investigating possibilities to keep its content accessible (either as a read-only access, or a migration of its content over here).

**Comment by KelsamGames** [comment]
I'll miss the Discourse forums, but Github Discussions is looking pretty good too. Looking forward to continuing to lurk everyone's work :P

**Reply by NemoKradXNA** [reply]
Yea, I'll miss it too (maybe as much as the old XNA forums), but this, for me at least is a much better format than the Discord server. Don't get me wrong, I love Discord, but not for this. 

As ever, thanks for setting up and facilitating this the MonoGame Team :)

**Reply by AristurtleDev** [reply]
One of the great things about moving here to discussions is when someone opens an issue that's really just a question, the issue can be converted to a discussion thread automatically, which helps keep the issue page cleaned up.

**Reply by Kwyrky** [reply]
For me it is the same. I think because I don't have that much free time available (basically almost none ^^). And to keep up with the pace in Discourse is not possible for me and it is everybody talking about everything in parallel. I think Discourse is good and helpful for the community but for people with little time for their hobby like me I think questions are better placed in a forum or now here where things are a bit slower :-)

**Reply by KelsamGames** [reply]
There seems to be a bit of confusion here. Discourse is the system the old forums uses (topics, replies, etc. similar to these GitHub Discussions). Discord is the chat system with a few channels, but with everyone just chatting live in one long thread in those channels.

**Reply by Kwyrky** [reply]
Yes you are right, I was in a hurry writing the response. I meant Discord and its fast pace.

**Comment by MrValentine7777** [comment]
Can we get a guide on usage or a brief explainer? [in a pinned thread of course]

**All hail the new forum!**

**Reply by NemoKradXNA** [reply]
There is a guide [here ](https://docs.github.com/en/site-policy/github-terms/github-community-guidelines)I believe. Or are you after something more specific?

**Reply by MrValentine7777** [reply]
Well, first thing is understanding how replies work, from an observation they work like X replies I guess? [briefly experienced it], this is a far better entry box, goodness was it a chore using the discourse one.

And the whole IsOpen by default in the search box does not tell you it is a search box immediately.

And why are only open threads the default? that makes it look like nothing has ever been discussed down the line... this was one of my concerns originally for someone familiar this is fine, but for the uninitiated first time viewers, it will look sparse on first impressions, any way to not have that defaulted?

And an explanation on those icons above the message window
```c#
So that people actually realise there is this functionality, 
instead of having to explain it to ever newcomer, 
we can just paste a link to a page/post.
Might want to also mention code blocks do not line wrap, so, yeah.
``` 

Just a few thoughts :) more as issues arise or solutions are required.

Happy to see this finally here though and so far, it feels like home.

Just trying to give feedback from a complete newbie viewpoint, many of us assume this stuff is straight forward, because we have the experience and mental capacity to navigate these realms, hope this helped.

**Reply by MrValentine7777** [reply]
And how do I disable the emails when I post something?

**Reply by MrValentine7777** [reply]
![image](https://github.com/MonoGame/MonoGame/assets/19196623/16123adc-cc41-423d-a1d7-51e16c62a21c)

This thing as mentioned above.

**Reply by MrValentine7777** [reply]
I checked settings in my account and there is nothing about discussions emails, only repos but not which...

**Reply by NemoKradXNA** [reply]
Ah, so a guide on GitHub Discussions in general then. :)

**Reply by MrValentine7777** [reply]
![image](https://github.com/MonoGame/MonoGame/assets/19196623/268f424f-8558-4ae2-9c1f-883480ba6d5a)

**Reply by MrValentine7777** [reply]
This might discourage people from using this mode of forum, already feeling annoyed myself honestly.

**Reply by NemoKradXNA** [reply]
For the email, do you not just Unsubscribe from the selected thread notifications?

![image](https://github.com/MonoGame/MonoGame/assets/2579338/9c8a78d2-a439-49f6-88d6-5fee15df9b3b)

**Reply by MrValentine7777** [reply]
See your email ping you just now?

**Reply by MrValentine7777** [reply]
About your own post here.

**Reply by NemoKradXNA** [reply]
Not since I unsubscribed to this thread :D

**Reply by MrValentine7777** [reply]
Defeats the point of the service lol, also, pages here do not auto update, a decade step back in QoS.

**Reply by NemoKradXNA** [reply]
I was only getting pings from others posting, but now I have unsubscribed I am not getting those either. Never did get notifications of my own posts.

**Reply by MrValentine7777** [reply]
So you could be replying to me, not have a clue I have posted and spend time wasted replying to something already solved.

**Reply by NemoKradXNA** [reply]
Actually I lie, just got that last one from you :/

**Reply by MrValentine7777** [reply]
https://github.com/MonoGame/MonoGame/discussions/8130#discussioncomment-8235193

Might be this then at the bottom..., can you check your notification in account settings if it is off?

**Reply by MrValentine7777** [reply]
Sorry, but this is incredibly stupid

![image](https://github.com/MonoGame/MonoGame/assets/19196623/ae562ef0-8896-4a69-b820-4701ba4fdb54)

**Reply by MrValentine7777** [reply]
I will take my time with this, feels broken, hopefully there are options for the host to play with.

**Reply by MrValentine7777** [reply]
When I refresh the page the link actually works instead of showing that daft popup...

EDIT

Putting this set down to teething problems and we will have to find ways around them, but overall, somewhat minor issues, but seeing 10+ emails a minute is frustrating and battery draining.

**Reply by NemoKradXNA** [reply]
Ah when I reply, it re subscribed me to the thread..... Sorry if this fires another email at you :P

**Reply by MrValentine7777** [reply]
Did you notice the closed thread yet?

**Reply by MrValentine7777** [reply]
![image](https://github.com/MonoGame/MonoGame/assets/19196623/153b6e40-31b3-42af-9a9f-c4861874013a)
this one

**Reply by MrValentine7777** [reply]
And it is an important message, hidden from view.

**Reply by MrValentine7777** [reply]
![image](https://github.com/MonoGame/MonoGame/assets/19196623/2079f101-abfb-4fb8-b8d4-bfc16ad94362)
This reminds me of Douglas Adams joke on the highway office hiding the details of the new motorway bypass in a basement with no access for the public.

**Reply by MrValentine7777** [reply]
Anyway hope these early insights help the team find solutions, and investigate them soon before we have an influx of people asking questions to things already answered.

**Reply by NemoKradXNA** [reply]
I am still at work, so not played much with it tbh :)

Have a look at this link for [git hub notifications](https://docs.github.com/en/account-and-profile/managing-subscriptions-and-notifications-on-github/managing-subscriptions-for-activity-on-github/managing-your-subscriptions), hoping I now wont get emails :/

**Reply by MrValentine7777** [reply]
Yeah, pretty much.

**Reply by MrValentine7777** [reply]
Quote reply is broken

**Reply by MrValentine7777** [reply]
https://github.com/MonoGame/MonoGame/discussions/8130#discussioncomment-8235187

Those were quote replies to that post

**Reply by NemoKradXNA** [reply]
[Or this one](https://github.com/settings/notifications)

**Reply by MrValentine7777** [reply]
https://github.com/MonoGame/MonoGame/discussions/8130#discussioncomment-8235491

Having to manually reference your message as quote reply is broken

That just silences the entire repo.

**Reply by MrValentine7777** [reply]
![image](https://github.com/MonoGame/MonoGame/assets/19196623/3f31c501-e4b6-44d3-a34c-c47953a400b8)

This is too broken for my liking...

And no I asked you about that one earlier.

There is no way to not get emails here without breaking everything else.

**Reply by MrValentine7777** [reply]
I shall leave those notes here and hope there are admin options to remedy some of the issues discovered.

**Reply by MrValentine7777** [reply]
But obviously I will have to unsubscribe now because emails.

**Reply by OtherStuffThatSpam** [reply]
hmm, I started to get too many emails now from here, I will also have to unsubscribe unless there is a config we can do.
